docs/database/database-spans.md

```diff
@@ -336,8 +343,16 @@ which SHOULD have low-cardinality and SHOULD NOT contain any dynamic or sensitiv
 > only contain information that has a significant impact on the query, database,
 > or application performance.
 
-Instrumentations that support query parsing SHOULD generate a query summary when
-one is not readily available from other sources.
+Instrumentation SHOULD set the query summary if it is readily available through instrumentation hooks or other sources.
```

Comment:
above:

> The `db.query.summary` attribute captures a shortened representation of a query text
which SHOULD have low-cardinality and SHOULD NOT contain any dynamic or sensitive data.

--> something like

> The `db.query.summary` attribute can be used to capture a shortened representation of the query text.
It SHOULD have low-cardinality and SHOULD NOT contain any dynamic or sensitive data.

Reply:
oh, also the section header

> Generating a summary of the query text

-->

> Generating a summary of the query

Reply:
nice! addressed in the https://github.com/open-telemetry/semantic-conventions/pull/2206/commits/4c4d1ccacfb29542ff7c8b100e2145f5ff3344ff